tests/Unit/PrestaShopBundle/Controller/ArgumentResolver/SearchParametersResolverTest.php

```diff
@@ -450,3 +455,10 @@ public static function getDefaults()
         ];
     }
 }
+
+interface SerializableTokenInterface extends TokenInterface
```

Comment:
Create a specific file for this.

Reply:
Is this really needed? This is only for this test and we already have another class in this file, right above this interface (`SampleFilters`).